docs/sources/operations/zone-ingesters.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Zone Aware Ingesters
```

Comment:
```suggestion
title: Zone aware ingesters
```

Reply:
why would we lowercase here?

Reply:
Because we've standardized on sentence case as our house Style (and I haven't gotten a chance to go through and standardize all the headings yet).

https://grafana.com/docs/writers-toolkit/style-guide/capitalization-punctuation/#sentence-case-capitalization